src/http_api_handler.erl

```diff
@@ -89,21 +89,27 @@ handle_request(<<"POST">>, _, json, Req, State) ->
             {true, Req2, State}
     end;
 
+handle_request(<<"DELETE">>, <<"/api/v1/contexts">>, json, Req, State) ->
+    ok = ergw_api:delete_contexts(length(ergw_api:contexts(all))),
```

Comment:
It might also be a good idea to have a `ergw_api:delete_contexts(all)` version.

The implementation of delete_contexts uses a random number to choose which context to delete. This is not needed if we really want to remove all contexts.

Reply:
Done